packages/syft/src/syft/core/tensor/smpc/utils.py

```diff
@@ -89,3 +94,31 @@ def get_ring_size(
         )
 
     return x_ring_size
+
+
+def count_wraps(share_list: List[np.ndarray]) -> np.ndarray:
```

Comment:
MAYBE: Could you also add a comment that this function is taken from CrypTen

Reply:
Done